File: Java/Pelanggan.java
public class Pelanggan {
    private int ID;
    private String Nama;
    private String No_Telephone;
    private String Alamat;

    // Constructor
    public Pelanggan(int id, String nama, String no_telp, String alamat) {
        this.ID = id;
        this.Nama = nama;
        this.No_Telephone = no_telp;
        this.Alamat = alamat;
    }

    // Getter ID
    public int getID() {
        return ID;
    }

    // Getter & Setter Nama
    public String getNama() {
        return Nama;
    }
    public void setNama(String nama) {
        this.Nama = nama;
    }

    // Getter & Setter No_Telephone
    public String getNo_Telephone() {
        return No_Telephone;
    }
    public void setNo_Telephone(String no_telp) {
        this.No_Telephone = no_telp;
    }

    // Getter & Setter Alamat
    public String getAlamat() {
        return Alamat;
    }
    public void setAlamat(String alamat) {
        this.Alamat = alamat;
    }
}

File: PHP/Pelanggan.php
<?php
class Pelanggan {
    private $ID;
    private $Nama;
    private $No_Telephone;
    private $Alamat;
    private $Photo;

    // Constructor
    public function __construct($ID, $Nama, $No_Telephone, $Alamat, $Photo) {
        $this->ID = $ID;
        $this->Nama = $Nama;
        $this->No_Telephone = $No_Telephone;
        $this->Alamat = $Alamat;
        $this->Photo = $Photo;
    }

    // Getter
    public function getID() { return $this->ID; }
    public function getNama() { return $this->Nama; }
    public function getNo_Telephone() { return $this->No_Telephone; }
    public function getAlamat() { return $this->Alamat; }
    public function getPhoto()  {return $this->Photo; }

    // Setter
    public function setNama($Nama) { $this->Nama = $Nama; }
    public function setNo_Telephone($No_Telephone) { $this->No_Telephone = $No_Telephone; }
    public function setAlamat($Alamat) { $this->Alamat = $Alamat; }
    public function setPhoto($Photo) {$this->Photo = $Photo;}
}
?>